iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift

```diff
@@ -137,6 +137,8 @@ class HomeViewController: UIViewController {
                 showAlert(with: "Error code: \(serverError.code)", title: "Server Error")
             case .couldNotBuildURLForRequest:
                 showAlert(with: "Error code: \(failure.code)", title: "Could Not Build URL")
+            case .missingAPIKey:
+                showAlert(with: "You need to make sure an API key is present", title: "Missing API Key")
```

Comment:
This is the kind of API Breaking change consumers will face. Our `switch failure {` will not compile as a new case is introduced. Xcode can however auto-fix the code, leave a placeholder for the new error to be handled (ie: consumer will add something like `showAlert` to handle error)